docs/asciidoc/modules/ROOT/pages/usage/index.adoc

```diff
@@ -11,11 +11,11 @@ But you can also integrate them into your Cypher statements which makes them so
 .Load JSON example
 [source,cypher,subs=attributes]
 ----
-WITH 'https://raw.githubusercontent.com/neo4j-contrib/neo4j-apoc-procedures/{branch}/src/test/resources/person.json' AS url
+WITH 'https://raw.githubusercontent.com/neo4j-contrib/neo4j-apoc-procedures/4.0/src/test/resources/person.json' AS url
```

Comment:
```suggestion
WITH 'https://raw.githubusercontent.com/neo4j-contrib/neo4j-apoc-procedures/core/{branch}/src/test/resources/person.json' AS url
```
this should work for 4.1, 4.2 and 4.3 (since Neo4j 4.0 has reached its End Of Life, no need to backport this fix to APOC 4.0)